internal/kafka/test/integration/cluster_mgr_test.go

```diff
@@ -140,13 +143,60 @@ func TestClusterManager_SuccessfulReconcile(t *testing.T) {
 	g.Expect(cluster.ExternalID).NotTo(gomega.Equal(""))
 	g.Expect(cluster.ExternalID).To(gomega.Equal(ocmCluster.ExternalID()))
 
+	// check that the default machine pool is created with correct min/max node and machine type.
+	// Only do the check for when running against real OCM environment
+	ocmConfig := test.TestServices.OCMConfig
+	if ocmConfig.MockMode != ocm.MockModeEmulateServer {
```

Comment:
Has the possibility of mocking the machinepool calls considered, to avoid having conditional code in the test?

Reply:
Yes. This required having to setup the mock response using the config objects which I finally deemed too much of a hassle to finally having to assert that the mock response was properly setup.